nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestElasticsearchObject.java

```diff
@@ -97,6 +97,36 @@ public void shouldRunIndexation() throws Exception {
         }
     }
 
+    @Test
+    public void shouldRunFlush() {
+        try (CloseableClientResponse response = httpClientRule.post("site/management/elasticsearch/flush", null)) {
+            assertEquals(SC_NO_CONTENT, response.getStatus());
```

Comment:
Indeed, why not return something more interesting, maybe the URL of a  GET endpoint to have the status?

Reply:
yes i was thinking to do something like that but the ES operation doesn't returns any id (at the beginning i was thinking that for each launched indexation i will have an id status), and i check the JSF admin i will find a way to returns the GET status on ES indexation.